test/tls/server.crt

```diff
@@ -0,0 +1,96 @@
+Certificate:
```

Comment:
We should not include this as it is highly likely that uses will literally copy-n-paste these files into their servers, rather than generate their own safe keys. I've seen this happen too many times.

I also highly recommend that we link to docs, rather than include them inline. This will avoid having the docs here get out of sync with the official docs.

I've released a v0.1.0 tag so it's easy to match up with a link:

https://github.com/prometheus/exporter-toolkit/blob/v0.1.0/https/README.md

Reply:
I will remove example and files and link to https://github.com/prometheus/exporter-toolkit/blob/v0.1.0/https/README.md directly, ok?

Reply:
Done!